files/at_tunnel/at_sprint_1.3.ifc

```diff
@@ -0,0 +1,51 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('','2023-03-16T15:32:15',(),(),'','0.1.0','');
+FILE_SCHEMA(('IFC4X4_A0BAE928'));
+ENDSEC;
+DATA;
+#1=IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#2=IFCDIRECTION((1.,0.,0.));
+#3=IFCDIRECTION((0.,1.,0.));
+#4=IFCDIRECTION((0.,0.,1.));
+#5=IFCCARTESIANPOINT((0.,0.,0.));
+#6=IFCTUNNEL('0nusg9yzb0APacqV3WDNTB',$,'TestTunnel',$,$,$,$,$,$,.RAILWAY.);
+#7=IFCPERSON('johannes',$,$,$,$,$,$,$);
+#8=IFCORGANIZATION($,'tbd',$,$,$);
+#9=IFCPERSONANDORGANIZATION(#7,#8,$);
+#10=IFCAPPLICATION(#8,'v1.0.0','IfcTunnelDeployment','1.0.0');
+#11=IFCOWNERHISTORY(#9,#10,$,.ADDED.,$,$,$,1678978354);
```

Comment:
If there is a ChangeAction (in your case .ADDED.), LastModifiedDate has to be set. Otherwise, ChangeAction can only be .NOCHANGE. or .NOTDEFINED.

Since both attributes are OPTIONAL, you could also leave them empty.